### PR TITLE
Pinterest and tumblr

### PR DESCRIPTION
**Pull request by prateekrastogi**
- Added pinterest and tumblr.

-  Also, added hashtag field in twitter, and title field in reddit.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
   "main": "dist/vanilla-sharing.umd.js",
   "module": "dist/vanilla-sharing.esm.js",
   "scripts": {
-    "precommit": "lint-staged",
+    "precommit": "npm run lint --",
```
[the remark this answers is not in the document]
Comment (prateekrastogi) [reply package.json]: At first, when i was trying to make my first commit on windows machine, npm was showing an error with change as suggested fix. The issue persisted on two different windows machine at home and work. So, I changed it. Now, I am unable to reproduce the error.  So, reverted to lint-staged.

File: README.md [hunk 2]
```diff
@@ -122,7 +123,33 @@ Share on Reddit
...
+  url: string,
+  description: string,
+  media: image url string,
+})
+```
+
+### `tumblr(options)`
+
+Share on Tumblr
+
```
[the remark this answers is not in the document]
Comment (prateekrastogi) [reply README.md]: attached link to documentation of both

**Comment by avdeev** [comment]
@prateekrastogi Thanks! I released it.

**Comment by prateekrastogi** [comment]
@avdeev Thanks. Just added the dependency in my project 😁

**Pull request merged**